### ISSUE TITLE
UE finder shows idle when the browser is not actually idle  

### ISSUE THREAD

**Issue by nedn** [catapult-project/catapult#2336]
The tracing categories are: "blink", "cc", "gpu", "renderer.scheduler", "toplevel", "v8", "cc.debug". The UE finder failing to infer the right UE here because of the missing data but it should show "cannot construct UE" instead of showing "idle".

@natduca @shubbie

**Comment by benshayden** [comment]
How should the UserModelBuilder determine whether the data is missing or there weren't any UEs?
Should/could GenerateTracingMetadataDict() save the enabled categories in the trace metadata? @vinterstum

**Comment by vinterstum** [comment]
Yeah I think we should just dump the whole TraceConfig into the metadata, to be honest. I'm kind of surprised we aren't already.

**Comment by nedn** [comment]
It already dump TraceConfig but we failed to add it to metadata:
![screenshot from 2016-05-24 13 27 53](https://cloud.githubusercontent.com/assets/911877/15518793/5dd26c24-21b3-11e6-93ba-43134d45b8ec.png)

**Comment by benshayden** [comment]
#1818

**Comment by vinterstum** [comment]
Adding metadata in https://codereview.chromium.org/2776753003

**Comment by vinterstum** [comment]
Landed in #459919
